src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/alert_request.cc

```diff
@@ -74,13 +74,12 @@ AlertRequest::~AlertRequest() {}
 bool AlertRequest::Init() {
   /* Timeout in milliseconds.
      If omitted a standard value of 10000 milliseconds is used.*/
+  uint32_t request_timeout_ = 0;
```

Comment:
@ValeriiMalkov why underscore in loca variables? Please retake this https://github.com/smartdevicelink/sdl_core/wiki/SDL-Coding-Style-Guide